addons/web_editor/static/src/js/backend.js

```diff
@@ -187,6 +187,18 @@ var FieldTextHtmlSimple = basic_fields.DebouncedField.extend({
         }
         return value;
     },
+
+    _renderTranslateButton: function () {
```

Comment:
I would keep the same API as in the TranslatableMixin, i.e. this function returns a jQuery node, and it's the responsibility of the caller to insert it at the right place (i'm not saying that this is a better design, but for the sake of consistency, i would do that).